### PR TITLE
Hotfix/details-about

### PR DESCRIPTION
**Pull request by gloriateodoro**
## Contexto

Esse PR faz ajustes na página /About da nova Home do Nossas

## Checklist
- [x] Retirar no momento o link de Baixar a auditoria
- [x] solucionar -> no mobile, quem financia o nossas está faltando o botão pra fazer uma doação e aumentar o espaçamento com o bloco de fundações.
- [x] solucionar -> linha do tempo: no mobile esse bloco está grudado na borda, ajustar o espaçamento lateral seguindo o padrão dos outros blocos
- [x] Linha do tempo: em 2020 adicionar Periferia que Faz
- [x] No modal com a bio, fechar o modal quando clicar fora dele também, não só no X
- [x] Liderança: Alessandra Orofino: na bio dela está faltando uma vírgula depois de "HBO"
- [x] Liderança: Cledisson: corrigir para Estrategista de Advocacy (ao invés de Head de Advocacy) na legenda da foto + no título da minibio
- [x] Equipe: Lívia Merlim: corrigir o sobrenome de Lívia, com "m" no final
- [x] About e Home -> Na mídia: Adicionar logo obama foundation e stanford
- [x] Linha do tempo: adicionar links das campanhas nos logos

### CODE REVIEW

**Comment by bnd-operator** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=nossas%2Fnossas-org).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/nossas/nossas-org/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by bnd-operator** [comment]
@gloriateodoro: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr | b87f8b2fea8a2703e3cf68c74a810dbfc7fff1b4 | [link](http://dashboard-jx.35.170.28.138.nip.io/namespaces/jx/pipelineruns/pr-5k5qt) | `/test pr`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Review by lpirola — APPROVED** [review]

**Comment by bnd-operator** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *[lpirola](https://github.com/nossas/nossas-org/pull/15#pullrequestreview-672579489 "Approved")*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=nossas%2Fnossas-org).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/nossas/nossas-org/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
